test/e2e/apimachinery/webhook.go

```diff
@@ -703,8 +705,369 @@ var _ = SIGDescribe("AdmissionWebhook [Privileged:ClusterAdmin]", func() {
+	})
+
+	/*
+		Release: v1.28
+		Testname: Validing Admission webhook, reject validating webhook configurations with invalid matchConditions
+		Description: Creates a validating webhook configuration with an invalid CEL expression in it's
+		matchConditions field. The api-server server should reject the create request with a "compilation
+		failed" error message.
+	*/
+	ginkgo.It("should reject validating webhook configurations with invalid match conditions", func(ctx context.Context) {
+		initalMatchConditions := []admissionregistrationv1.MatchCondition{
+			{
+				Name:       "invalid-expression-1",
+				Expression: "... [] bad expression",
+			},
+		}
+
+		ginkgo.By("creating a validating webhook with match conditions")
+		validatingWebhookConfiguration := newValidatingWebhookWithMatchConditions(f, servicePort, certCtx, initalMatchConditions)
+
+		_, err := createValidatingWebhookConfiguration(ctx, f, validatingWebhookConfiguration)
+		framework.ExpectError(err, "create validatingwebhookconfiguration should have been denied by the api-server")
+		expectedErrMsg := "compilation failed"
+		gomega.Expect(strings.Contains(err.Error(), expectedErrMsg)).To(gomega.BeTrue())
+	})
+
+	/*
+		Release: v1.28
+		Testname: Mutating Admission webhook, reject mutating webhook configurations with invalid matchConditions
+		Description: Creates a mutating webhook configuration with an invalid CEL expression in it's
+		matchConditions field. The api-server server should reject the create request with a "compilation
+		failed" error message.
+	*/
+	ginkgo.It("should reject mutating webhook configurations with invalid match conditions", func(ctx context.Context) {
+		initalMatchConditions := []admissionregistrationv1.MatchCondition{
+			{
+				Name:       "invalid-expression-1",
+				Expression: "... [] bad expression",
+			},
+		}
+
+		ginkgo.By("creating a mutating webhook with match conditions")
+		mutatingWebhookConfiguration := newMutatingWebhookWithMatchConditions(f, servicePort, certCtx, initalMatchConditions)
+
+		_, err := createMutatingWebhookConfiguration(ctx, f, mutatingWebhookConfiguration)
+		framework.ExpectError(err, "create mutatingwebhookconfiguration should have been denied by the api-server")
+		expectedErrMsg := "compilation failed"
+		gomega.Expect(strings.Contains(err.Error(), expectedErrMsg)).To(gomega.BeTrue())
+	})
+
+	/*
+		Release: v1.28
+		Testname: Validating Admission webhook, validating webhook exclude leases using match conditions field.
+		Description: Create a validating webhook configuration with matchConditions field that
+		will reject all resources except the coordination.k8s.io/lease ones. Try to create pods
+		until the webhook is ready and rejecting the pods with "denied" error message. Create
+		a Lease object and validate that it bypasses the webhook. Create a configMap and validate
+		that it's rejected by the webhook.
+	*/
```

Comment:
e2e test that are disruptive and affect or possible affect other e2e test running at the same in the cluster must me marked as `[Serial]` and `[Disruptive]`, is this denying everything in the cluster or just in the same namespace?